repo/packages/S/spark/0/mantl.json

```diff
@@ -0,0 +1,5 @@
+{
+  "mantl": {
+    "load-balancer": "external"
```

Comment:
I see your points. Now it defaults to "off". I still like to be able to show the Spark UI when I explain things to my collaborators. It is a nice way to get their attention, since they are used to that. Is there any way to change this setting when I install the package through the mant-api, via REST call?